composer.json

```diff
@@ -65,16 +65,16 @@
     "require-dev": {
         "dama/doctrine-test-bundle": "^v6.7",
         "ibexa/ci-scripts": "^0.2@dev",
-        "ibexa/behat": "~5.0.0@dev",
+        "ibexa/behat": "~5.0.x-dev",
         "friendsofphp/php-cs-fixer": "^3.0",
         "phpunit/phpunit": "^9.5",
         "matthiasnoback/symfony-dependency-injection-test": "^4.0",
-        "ibexa/doctrine-schema": "~5.0.0@dev",
-        "ibexa/http-cache": "~5.0.0@dev",
+        "ibexa/doctrine-schema": "~5.0.x-dev",
+        "ibexa/http-cache": "~5.0.x-dev",
         "ibexa/code-style": "^1.3.x-dev",
-        "ibexa/notifications": "~5.0.0@dev",
+        "ibexa/notifications": "~5.0.x-dev",
         "ibexa/test-rest": "^0.1.x-dev",
-        "ibexa/test-core": "^0.1.x-dev",
+        "ibexa/test-core": "~5.0.x-dev",
```

Comment:
Is this correct?

Reply:
should be, but ibexa/test-rest also will be bumped by @alongosz

Reply:
> Is this correct?

Yes. Fixing test-rest, which is the root cause of the issue.